.env

```diff
@@ -0,0 +1 @@
+REACT_APP_BASE_URL=http://illinkchemipasseràleo
```

Comment:
cos'è questo?

Reply:
Oh sorry, env variable ahahah but since I didn't have it yet I wrote a silly placeholder